packages/components/src/slot-fill/index.js

```diff
@@ -3,11 +3,11 @@
  */
 import Slot from './slot';
 import Fill from './fill';
-import Provider from './provider';
+import Provider, { Consumer } from './provider';
```

Comment:
Should we rename it to context and export both components the same way? This could be also simplified to:
```js
export { SlotFillProvider as Provider, Consumer } from './context';
```

Reply:
Actually, we don't need to export the `Consumer` directly because we're already exporting specialized consumers (`Slot`, and `Fill`). Removed it.